### DISCUSSION TITLE
need help wit sound

### DISCUSSION THREAD

**Discussion by monkey-d-uchiha1** [jef/streetmerchant#1774] [Help] [answered]
error is this anyone know how to fix please
[3:11:46 PM] warn :: ✖ couldn't find sound player

**Comment by monkey-d-uchiha1** [comment, marked as answer]
found fix

**Reply by yumotto** [reply]
What did you do to fix this? I have this same issue. I also believe it's not reading my sound file path correctly. Id appreciate it if you could show me your .env file configuration. Thanks in advance.
